pinc/Project.inc

```diff
@@ -634,7 +634,12 @@ class Project
         }
 
         // Create or update the metadata file
-        $this->generate_metadata_json();
+        try {
+            $this->generate_metadata_json();
```

Comment:
would it make more sense to check in the `generate_metadata_json` method?

Reply:
We actually raise the exception in that function, I did it this way so that the function wouldn't silently fail.